rosdep/python.yaml

```
@@ -5014,6 +5014,12 @@ python3-gnupg:
   fedora: [python3-gnupg]
   gentoo: [dev-python/python-gnupg]
   ubuntu: [python3-gnupg]
+python3-ifcfg:
+  debian:
+    buster: [python3-ifcfg]
+    stretch: [python3-ifcfg]
+  ubuntu:
```

I made this package available in Fedora and EPEL: https://apps.fedoraproject.org/packages/python-ifcfg

```suggestion
  fedora: [python3-ifcfg]
  rhel: ['python%{python3_pkgversion}-ifcfg']
  ubuntu:
```